### DISCUSSION TITLE
Remote Access

### DISCUSSION THREAD

**Discussion by hojuguy** [hassio-addons/addon-plex#199] [Q&A]
Hi, first-time user and HA novice, so please bare with me.

I am running my Plex installed on my Home Assistant Server. For some reason though, I cannot remotely access my Plex server. Not sure if it matters, but I have my HA running through Cloudflare -  but currently connected locally.

I can access my Plex Server by going to [http://192.168.##.##:32400](http://192.0.0.168/##.##:32400) (locally) but when I try to port forward that IP and port, it doesn't work. When checking the remote access page, it says the private IP is 172.30.##.## with a port of 32400. I tried port forwarding that IP address as well no luck. TBH, I struggle to understand computer networks but I would have thought my server IP would be the same as the Private IP in the remote access section.

Any assistance would be muchly appreciated as I have been googling for weeks without success.
![image](https://github.com/hassio-addons/addon-plex/assets/104206218/b644fdde-27e3-4592-8586-7e45a250f7f4)
![image](https://github.com/hassio-addons/addon-plex/assets/104206218/cb98296f-9ef7-4f39-b4b1-a82a1bd60f27)

**Comment by asimov44** [comment]
I have the same problem. Someone knows where is the .config files for Plex Server? Thank_

**Reply by frenck** [reply]
You can't and should run plex through cloudflare. It is now allowed in their ToS.

**Reply by asimov44** [reply]
The problem is that Plex server detect as local ip a 172.x..x.x, but the real ip is under rango 192.168.x.x
 I think this is becase plex server is running in a docker install.

**Reply by frenck** [reply]
That is documented. Please read the add-on documentation.

**Reply by asimov44** [reply]
I don´t find this point. Please, can you share the link where is this information? Thank you a lot.!

**Reply by frenck** [reply]
In the documentation: https://github.com/hassio-addons/addon-plex/blob/main/plex/DOCS.md#solving-connection-issues-with-plex

**Reply by asimov44** [reply]
Sorry, I see this item in the documentation, I try it, but the problem still occur. I think the solution is find the configuration file config.yaml


---
name: Plex Media Server
version: dev
slug: plex
description: Recorded media, live TV, online news, and podcasts ready to stream.
url: https://github.com/hassio-addons/addon-plex
codenotary: codenotary@frenck.dev
webui: http://[HOST]:[PORT:32400]/web  <<<<<< HERE CHANGE HOST by the ip by my home assistan server, because in automatic it take a host address WRONG  172.xxxxx LIKE you can see in the pictures above.
startup: services
init: false
arch:
  - aarch64
  - amd64
  - armv7
map:
  - media:rw
  - share:rw
  - ssl
devices:
  - /dev/dri
  - /dev/dri/card0
  - /dev/dri/card1
  - /dev/dri/renderD128
  - /dev/vchiq
ports:
  3005/tcp: 3005
  8324/tcp: 8324
  32400/tcp: 32400
  32410/udp: 32410
  32412/udp: 32412
  32413/udp: 32413
  32414/udp: 32414
  32469/tcp: 32469
ports_description:
  3005/tcp: Plex Home Theater via Plex Companion
  8324/tcp: Plex for Roku via Plex Companion
  32400/tcp: Plex Media Server Interface
  32410/udp: GDM network discovery
  32412/udp: GDM network discovery
  32413/udp: GDM network discovery
  32414/udp: GDM network discovery
  32469/tcp: Plex DLNA Server
hassio_api: true
backup_exclude:
  - "**/Plex Media Server/Cache/**"
  - "**/Plex Media Server/Plug-in Support/Caches/**"
  - "**/Plex Media Server/Logs/**"
  - "**/Plex Media Server/Crash Reports/**"
  - "**/Plex Media Server/Diagnostics/**"
options:
  claim_code: ""
schema:
  log_level: list(trace|debug|info|notice|warning|error|fatal)?
  claim_code: str


Thank, again for your interesting in my problem. Very thank to you.

**Reply by frenck** [reply]
This has nothing to do with the sources of the add-on. Please see the add-on documentation.

**Reply by asimov44** [reply]
Hi. The problem is the instalation of plexserver make a /etc/resolv.conf like this


# Generated by Docker Engine.
# This file can be edited; Docker Engine will not make further changes once it
# has been modified.

nameserver 172.30.32.3
search local.hass.io
options timeout:10

# Based on host file: '/etc/resolv.conf'
# Overrides: [nameservers search options]

I think

**Reply by frenck** [reply]
That isn't relevant, as this is a client send option.

You really need to follow the docs.

**Comment by madbrain76** [comment]
I followed the docs, but I think it's in need of some clarifications.

First, in the installation section, in step 8, it should be made clear that "the plex admin interface" means the local one at http://homeassistant.local:32400 . Unless that's not what was intended ?

Then, under the "Solving connection issues with Plex", it says to login to the Plex web interface. Does it mean plex.tv ?

Finally, if so, I went to setting, but there is no "server tab" for me to click. I do see a tab called "Home Assistant", however, which presumably in the Plex media server from the add-on. Is this what I'm meant to click on ? This is probably it, since

**Reply by asimov44** [reply]
Finally the solution, I had to open the ports in the internet router and assign the home assistant ip 192.XXx and the TCP port assigning a specific one. For the other Plex servers that I have at home directly in debian it is not necessary to do this because the address 172.XXX does not appear in the configuration of the Plex server. Anyway.

**Comment by asimov44** [comment]
> Hi, first-time user and HA novice, so please bare with me.
> 
> I am running my Plex installed on my Home Assistant Server. For some reason though, I cannot remotely access my Plex server. Not sure if it matters, but I have my HA running through Cloudflare - but currently connected locally.
> 
> I can access my Plex Server by going to [http://192.168.##.##:32400](http://192.0.0.168/##.##:32400) (locally) but when I try to port forward that IP and port, it doesn't work. When checking the remote access page, it says the private IP is 172.30.##.## with a port of 32400. I tried port forwarding that IP address as well no luck. TBH, I struggle to understand computer networks but I would have thought my server IP would be the same as the Private IP in the remote access section.
> 
> Any assistance would be muchly appreciated as I have been googling for weeks without success. ![image](https://private-user-images.githubusercontent.com/104206218/249703487-b644fdde-27e3-4592-8586-7e45a250f7f4.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NDA1MDkyMDgsIm5iZiI6MTc0MDUwODkwOCwicGF0aCI6Ii8xMDQyMDYyMTgvMjQ5NzAzNDg3LWI2NDRmZGRlLTI3ZTMtNDU5Mi04NTg2LTdlNDVhMjUwZjdmNC5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjUwMjI1JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI1MDIyNVQxODQxNDhaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT04MzIxMjFkMDczM2YyOTZiNGUwMGRmOWVlNDFkMTc1YTc0MjNmZTA5YjQ2YzM0NjM4MDg0MjEyMjU5MTM0ZGY4JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.Rl-OHf3YjbnoaWcZXEpRp936V69zV53BsNUH5w4dcW8) ![image](https://private-user-images.githubusercontent.com/104206218/249703593-cb98296f-9ef7-4f39-b4b1-a82a1bd60f27.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NDA1MDkyMDgsIm5iZiI6MTc0MDUwODkwOCwicGF0aCI6Ii8xMDQyMDYyMTgvMjQ5NzAzNTkzLWNiOTgyOTZmLTllZjctNGYzOS1iNGIxLWE4MmExYmQ2MGYyNy5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjUwMjI1JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI1MDIyNVQxODQxNDhaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1kNjg3MjFlZTQyMWM5NWRjZGI5Y2Y3ZTJlYTZkMDE2NGRmMDczM2M3NTg1NWJmNWE0Y2RjOTBiNDBmY2RhMmI0JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.wYADAfs93NCyl8_XMj8kI6_dgZzqmp29DeMmcq1SU8I)
